docs/swarm/ingress.md

```diff
@@ -0,0 +1,119 @@
+<!--[metadata]>
++++
+title = "Use swarm mode routing mesh"
+description = "Use the routing mesh to publish services externally to a swarm"
+keywords = ["guide", "swarm mode", "swarm", "network", "ingress", "routing mesh"]
+[menu.main]
+identifier="ingress-guide"
+parent="engine_swarm"
+weight=17
++++
+<![end-metadata]-->
+
+# Use swarm mode routing mesh
+
+Docker Engine swarm mode makes it easy to publish ports for services to make
+them available outside the swarm. All nodes participate in an ingress
+**routing mesh**. The routing mesh enables each node in the swarm to accept
+connections on published ports for any service running in the swarm, even if
+there's no task running on the node. The internal load balancer routes all
```

Comment:
Technically, its the `routing mesh` (which is made of Load-Balancer and Overlay network) that routes the incoming requests. The Load-Balancer just identifies a backend-ip to reach and the overlay `ingress` network routes the traffic to the right container.